src/api.js

```diff
@@ -48,3 +48,15 @@ export const updateVotes = (review_id, vote) => {
       return review;
     });
 };
+
+export const fetchCommentsByReviewId = (review_id) => {
```

Comment:
I would ditch fetch for Axios if I were you - it's easier and better. Up to you though!